.vscode/tasks.json

```diff
@@ -0,0 +1,64 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Configure Waf for Debug 32-bit",
+            "type": "shell",
+            "command": "./waf configure -T debug --prefix=out/",
+            "problemMatcher": [],
+            "windows": {
+                "command": "./waf.bat configure -T debug --prefix=out/"
+            },
+            "osx": {
+                "command": "python3 waf configure -T debug --prefix=out/"
+            }
+        },
+        {
+            "label": "Configure Waf for Debug 64-bit",
+            "type": "shell",
+            "command": "./waf configure -T debug --64bits --prefix=out/",
+            "problemMatcher": [],
+            "windows": {
+                "command": "./waf.bat configure -T debug --64bits --prefix=out/"
+            },
+            "osx": {
+                "command": "python3 waf configure -T debug --64bits --prefix=out/"
+            }
+        },
+        {
+            "label": "Configure Waf for Debug on outdated Android",
+            "type": "shell",
+            "command": "./waf configure -T release --android=armeabi-v7a-hard,4.9,21",
```

Comment:
Label says that it's debug build but it configures for release
Also I think you should remove "outdated". Old NDK allows engine to run on almost any Android version

Reply:
It is indeed outdated, as it runs on an unsupported version.
Any supported version of the NDK would be preferable for its ease of installation, and ability to get bug fixes and security updates, so ideally we should also have a build task for that too. Of course, support would have to be added to the code-base as well so it doesn't confuse new contributors.
In this situation, we could have one task named `Configure Waf for Debug on Android (NDK r10e)` and one named `Configure Waf for Debug on Android (NDK <lowest LTS>)`.

I'll just go ahead and specify the NDK version, instead of calling it "outdated" for now.